dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java

```diff
@@ -139,7 +146,7 @@ public String getEndpointsInfo() {
      */
     private static class EndpointLogger {
         private final String rootPath;
-        private final List<String> endpoints = Lists.newArrayList();
+        private final List<EndpointLogLine> endpointLogLines = Lists.newArrayList();
```

Comment:
Could we pass a link to the root collection, rather then creating a new collection for every resource?
Now this collecton is created only to be captured.